server/src/main/java/umm3601/user/UserController.java

```diff
@@ -115,22 +119,29 @@ public void getUsers(Context ctx) {
    */
   public void addNewUser(Context ctx) {
     User newUser = ctx.bodyValidator(User.class)
-      .check(usr -> usr.name != null && usr.name.length() > 0, "User must have a non-empty user name") //Verify that the user has a name that is not blank
-      .check(usr -> usr.email.matches(emailRegex), "User must have a legal email") // Verify that the provided email is a valid email
-      .check(usr -> usr.age > 0, "User's age must be greater than zero") // Verify that the provided age is > 0
-      .check(usr -> usr.role.matches("^(admin|editor|viewer)$"), "User must have a legal user role") // Verify that the role is one of the valid roles
-      .check(usr -> usr.company != null && usr.company.length() > 0, "User must have a non-empty company name") // Verify that the user has a company that is not blank
+       //Verify that the user has a name that is not blank
```

Comment:
Quick whitespace fix

```suggestion
       // Verify that the user has a name that is not blank
```